src/components/CampaignForm.tsx

```diff
@@ -163,7 +163,7 @@ const CampaignForm = ({ onSubmit, onCancel, campaign }: CampaignFormProps): JSX.
     const handleSubmit = (values: Record<string, string>) => {
         const { info_text, status, title, visibility, manager_id } = values;
         onSubmit({
-            ...info_text ? { info_text } : null,
+            info_text: info_text == undefined ? '' : info_text,
```

Comment:
good workaround, this should be an issue in zetkin platform so it can be fixed in the API. Unless there's good reason for it to demand an empty string instead of `null`

Reply:
Please create an issue @nehahirve

Reply:
Sorry resolved this cuz I made a change so it's better logic thanks to @richardolsson's recommendation